integration/command.go

```diff
@@ -113,9 +113,9 @@ func DeployInspektorGadget(image, imagePullPolicy string) *Command {
 
 func DeploySPO(limitReplicas, patchWebhookConfig, bestEffortResourceMgmt bool) *Command {
 	cmdStr := fmt.Sprintf(`
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.8.0/cert-manager.yaml
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.10.1/cert-manager.yaml
```

Comment:
Why v1.10.1? The [documentation](https://github.com/kubernetes-sigs/security-profiles-operator/blob/v0.6.0/installation-usage.md#install-operator) suggests 1.10.0.

Reply:
Ah I was checking `main` where they already changed to `v1.10.1`.